src/renderer/features/network/NetworkSelector/ui/NetworkSelector.tsx

```diff
@@ -95,43 +96,40 @@ export const NetworkSelector = ({
                           <FootnoteText className="text-text-secondary truncate">
                             {Title[type](t, node?.name)}
                           </FootnoteText>
-                          {node?.url && <HelpText className="text-text-tertiary truncate">{node.url}</HelpText>}
+                          <HelpText className="text-text-tertiary truncate">{node?.url}</HelpText>
```

Comment:
if there were no node or url, we can have 'undefined' in this HelpText block

Reply:
`if (!children) return null;` inside `TextBase`